hack/upload-buildpacks.sh

```diff
@@ -18,13 +18,26 @@ set -eu
 
 scriptpath=$(cd $(dirname $0)/.. && pwd)
 
-# Upload stacks
-PUBLISH=true $scriptpath/hack/upload-buildpacks-stack.sh
+# Set the container registry
+# If KF_REGISTRY is populated then use that otherwise try setting it to gcr
+function setregistry {
```

Comment:
Why break this out into a function when it's only used once?

Reply:
It is evident that we would have to refactor this bit to make it fully generic. If we break things down now, then it might be easier to port some things over to some sort of utils.

Reply:
Sounds good. Thanks.